Branch-SDK/Branch-SDK/Requests/BranchOpenRequest.m

```diff
@@ -129,7 +130,12 @@ - (void)processResponse:(BNCServerResponse *)response error:(NSError *)error {
     if (data[BRANCH_RESPONSE_KEY_BRANCH_IDENTITY]) {
         preferenceHelper.identityID = data[BRANCH_RESPONSE_KEY_BRANCH_IDENTITY];
     }
-    
+   
+    NSArray *promoViewArray = [[[Branch getInstance] getLatestReferringParams] objectForKey:@"app_promo_data"];
```

Comment:
shouldn't this just be `NSArray *promoViewArray = data[@"app_promo_data"];`? Also, shouldn't we make `@"app_promo_data"` a constant somewhere?